modules/dnn/src/op_webnn.hpp

```diff
@@ -31,6 +31,66 @@ constexpr bool haveWebNN() {
 #endif
 }
 
+#ifdef HAVE_WEBNN
+
+class WebNNBackendNode;
+
+
+class WebNNGraph
+{
+public:
+    WebNNGraph();
+
+    bool isInitialized();
+
+    void init(Target targetId);
+
+    void forward(const std::vector<Ptr<BackendWrapper> >& outBlobsWrappers);
+
+    void createGraph(Target targetId);
+
+private:
+    ml::GraphBuilder builder;
+    ml::Context mContext;
+    ml::Graph mGraph;
+    ml::NamedResults mResults;
+    bool isInit = false;
+};
+
+class WebNNBackendNode : public BackendNode
+{
+public:
+    WebNNBackendNode(const std::vector<Ptr<BackendNode> >& nodes, Ptr<Layer>& layer,
+                        std::vector<Mat*>& inputs, std::vector<Mat>& outputs,
+                        std::vector<Mat>& internals);
+
+    void setName(const std::string& name);
+
+    std::shared_ptr<ml::Operand> operand;
```

Comment:
ml::Operand itself is reference counted, so it would be fine just `ml::Operand operand`.

Reply:
Fixed.